services/workflows-service/src/app.module.ts

```diff
@@ -62,6 +62,9 @@ import { SessionAuthMiddleware } from '@/auth/session-auth.middleware';
 })
 export class AppModule {
   configure(consumer: MiddlewareConsumer) {
-    consumer.apply(SessionAuthMiddleware).forRoutes('internal/*');
+    consumer.apply(SessionAuthMiddleware).forRoutes({
+      path: 'v(\\d)+/internal/*',
+      method: RequestMethod.ALL,
```

Comment:
can you explain this change?

Reply:
It applies the middleware on any API version endpoint. It's a regex, v*/internal/*, for example, v3/internal/example